src/Common/Api.ts

```diff
@@ -150,7 +150,11 @@ async function fetchAPI<K = object>(
                     })
                 } else {
                     handleLogout()
-                    return { code: 401, status: 'Unauthorized', result: [] }
+                    return await new Promise((resolve) => {
+                        setTimeout(() => {
```

Comment:
using setTimeout is this expected?

Reply:
Yes, added comment here for better documentation